### PR TITLE
Update kinematics config to allow head to search

### PR DESCRIPTION
**Pull request by tayloryoung6396**

### CODE REVIEW

File: module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml [hunk 1]
```diff
@@ -26,8 +26,8 @@ head:
     base_position_from_origin: [-0.007, 0, 0.21] # [x, y, z]
   # head movement limits
   limits: # pitch limit is a function of yaw ... need to calculate this properly
-    yaw: [-pi/4, pi/4] # [min_yaw, max_yaw]
-    pitch: [pi/6, pi/6] # [min_pitch, max_pitch]
+    yaw: [-pi/3, pi/3] # [min_yaw, max_yaw]
+    pitch: [-pi/5, pi/5] # [min_pitch, max_pitch]
```
Comment (KipHamiltons) [inline module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml:30]: Could you please explain this change?

**Review by KipHamiltons — COMMENTED** [review]
Seems fine, but one question

**Comment by KipHamiltons** [comment]
Is this change still required @tayloryoung6396?

File: module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml (hunk 1 above)
Comment (tayloryoung6396) [reply module/motion/KinematicsConfiguration/data/config/KinematicsConfiguration.yaml:30]: This is needed to allow the head to move

**Review by tayloryoung6396 — COMMENTED** [review]
@KipHamiltons This is needed, master still doesn't allow the head to move

**Comment by KipHamiltons** [comment]
You said you were changing this, then never pushed changes @tayloryoung6396. What's happening with this branch/PR?
Also, where does this fit in with what @NDxiura is doing?

**Comment by KipHamiltons** [comment]
I would like to see this branch revived or removed please @tayloryoung6396

**Comment by KipHamiltons** [comment]
#581 Addresses the same problems and is more developed.

**Pull request closed without merging**